uber/site_sections/groups.py

```diff
@@ -1,4 +1,101 @@
 from uber.common import *
+from uber.custom_tags import pluralize
+
+
+_entangled_ribbons = set(
+    getattr(c, r, -1) for r in ['BAND', 'DEPT_HEAD_RIBBON', 'PANELIST_RIBBON'])
+
+
+def _is_attendee_disentangled(attendee):
+    """
+    Returns True if the attendee has an unpaid badge and does not have any
+    other roles in the system.
+    """
+    return attendee.paid not in [c.HAS_PAID, c.NEED_NOT_PAY] \
```

Comment:
You should add ``c.REFUNDED`` to this list, because a lot of Panelists and MITS presenters and such have already had their badges refunded.  (Admittedly, I'd expect those cases to be caught by one of the other checks so this safeguard probably won't matter in practice.)